_posts/2018-04-26-IAM-managed-SSH-users.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: post
+title: Managing EC2 SSH users with IAM
+banner: /assets/posts/2018-04-26-iam-managed-ssh-users/iamec2.png
+author: lvandonkersgoed
+---
+
+Servers in the cloud are supposed to be immutable and stateless. But what if the users allowed on a system are dynamic? You need a management system for your users!
+
+At Sentia we've learned to treat our server instances as [cattle](http://cloudscaling.com/blog/cloud-computing/the-history-of-pets-vs-cattle/). They should be replaceable at any time, which is achieved through statelessness, and every server in an autoscaling group should be the same, which is achieved by using immutable sources (like an AMI) for every instance.
+
+![Cattle](../../../assets/posts/2018-04-26-iam-managed-ssh-users/cattle.jpg)
+
+
+Managing users the old way
+--------------------------
+Most environments we build and maintain are based on Linux. In general, these systems have a limited amount of users; because of security and operability reasons, we do not often allow our customers to login to instances.
+
+In the few cases where the customer needs and is allowed SSH access to his systems, user management is easy. In our AMI creation process, we define the username(s) the customer can use, add a user in our automated AMI creation procedure and include their public SSH key in the `authorized_keys` file.
```

Comment:
I would change "customers" to "clients" everywhere you use it. Customers sounds like someone at the local butcher ;)